solidity/contracts/libs/RateLimited.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+/**
+ * @title RateLimited
+ * @notice A contract used to keep track of an address sender's token amount limits
+ **/
+contract RateLimited is OwnableUpgradeable {
+    uint256 public constant DURATION = 1 days; // 86400
+
+    mapping(address sender => Limit limit) public limits;
```

Comment:
I wonder if we could allow the owner of the sender/recipient configure the limits? instead of one per rate limit contract

Reply:
alternatively, we can remove the mapping altogether
h/t @nambrot

Reply:
Removed mapping